desktop/config.coffee

```diff
@@ -97,6 +97,7 @@ module.exports =
   SITEMAP_BASE_URL: 'http://artsy-sitemaps.s3-website-us-east-1.amazonaws.com'
   STRIPE_PUBLISHABLE_KEY: null
   TEAM_BLOGS: '^\/life-at-artsy$|^\/artsy-education$|^\/gallery-insights$'
+  TARGET_CAMPAIGN_URL: '/seattle-art-fair-2017'
```

Comment:
I did have one question. Should this value be set to null in the config file? I'm going to set them as environment variables regardless, but I was wondering would it be more appropriate to it to null here?